dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/collection/CollectionUtils.java

```diff
@@ -269,4 +270,51 @@ public static <T> boolean addIfNotNull( Collection<T> collection, T object )
 
         return false;
     }
+
+    /**
+     * Returns a map of 11 key/value pairs.
+     *
+     * @param k1 the first mapping's key
+     * @param v1 the first mapping's value
+     * @param k2 the second mapping's key
+     * @param v2 the second mapping's value
+     * @param k3 the third mapping's key
+     * @param v3 the third mapping's value
+     * @param k4 the fourth mapping's key
+     * @param v4 the fourth mapping's value
+     * @param k5 the fifth mapping's key
+     * @param v5 the fifth mapping's value
+     * @param k6 the sixth mapping's key
+     * @param v6 the sixth mapping's value
+     * @param k7 the seventh mapping's key
+     * @param v7 the seventh mapping's value
+     * @param k8 the eighth mapping's key
+     * @param v8 the eighth mapping's value
+     * @param k9 the ninth mapping's key
+     * @param v9 the ninth mapping's value
+     * @param k10 the tenth mapping's key
+     * @param v10 the tenth mapping's value
+     * @param k11 the eveventh mapping's key
+     * @param v11 the eleventh mapping's value
```

Comment:
I think these don't provide much - should be pretty clear without

Reply:
Agreed. See [pull/11717](https://github.com/dhis2/dhis2-core/pull/11717). (I merged this PR to be sure it made it before the 2.39 feature freeze.)